rules/policy.go

```diff
@@ -111,7 +106,29 @@ func (r *DefaultRuleRenderer) ProtoRuleToIptablesRules(pRule *proto.Rule, ipVers
 	if filteredAll {
 		return nil
 	}
+	return &ruleCopy
+}
+
+func (r *DefaultRuleRenderer) ProtoRuleToIptablesRules(pRule *proto.Rule, ipVersion uint8) []iptables.Rule {
+	// Filter the CIDRs to the IP version that we're rendering.  In general, we should have an
```

Comment:
nit: In golang, for the purpose of documentation, it is common (linter may - like in my vi - warn about that) that exported functions are prepended with a comment that starts `// <fucn name>` and explains what the function does. It can be then rendered by godoc.

```
// MyCoolFunction is cool
func MyCoolFunction () {
    ...
}
```

Seems like this comment falls in that category.

The same holds for exported types, variables, etc.

Reply:
Agree, code would be improved by moving the comment to the new function.

Here, all the reader needs to know is that a rule may not match the IP version that we're rendering for and hence we may filter it out.  Don't even need a comment for that, the code is very clear.

However, the algorithm is worth explaining in the new function since it's a bit subtle.